### PR TITLE
Add drafter to workflows

### PR DESCRIPTION
**Pull request by kpodsiad**
[Drafter readme](https://github.com/release-drafter/release-drafter#readme)

### CODE REVIEW

File: .github/release-drafter-config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - title: '🚀 Features'
+    labels:
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bug'
+autolabeler:
+  - label: 'documentation'
+    files:
+      - '*.md'
```
Comment (lwronski) [inline .github/release-drafter-config.yml:13]: It's a good idea to include all the files in the docs directory.

File: .github/release-drafter-config.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    branch:
+      - '/docs{0,1}\/.+/'
```
Comment (lwronski) [inline .github/release-drafter-config.yml:15]: I don't think I need this option, we don't have this naming for branch

File: .github/release-drafter-config.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - label: 'bug'
+    branch:
+      - '/fix\/.+/'
```
Comment (lwronski) [inline .github/release-drafter-config.yml:18]: How about the naming branch with prefix `bugfix`?

**Review by lwronski — COMMENTED** [review]
We should standardize branch names, my proposition
- branch with bugs `'/bugfix\/.+/'`
- branch with feature `'/feature\/.+/'`

File: .github/release-drafter-config.yml (hunk 1 above)
Comment (kpodsiad) [reply .github/release-drafter-config.yml:13]: I think that for the autolabelel purpose `.md` plus `.svg` should be good enough. Currently, our docs consist of only md and svg files

File: .github/release-drafter-config.yml (hunk 2 above)
Comment (kpodsiad) [reply .github/release-drafter-config.yml:15]: Yeah, true. For docs, file matcher should be enough

File: .github/release-drafter-config.yml (hunk 1 above)
Comment (lwronski) [reply .github/release-drafter-config.yml:13]: Hope you will remember that :) 
Yes, it is good enough for now, but later there will be files with other extension.

**Review by lwronski — APPROVED** [review]
When CI is finish successfully, you can merge.

**Comment by kpodsiad** [comment]
> When CI is finish successfully, you can merge.

According to the https://github.com/release-drafter/release-drafter/issues/398 config file should be on the default branch, otherwise workflow will fail. So I'm gonna transplant the config file into main and then check if workflow passes.

**Pull request merged**